lib/user.rb

```diff
@@ -0,0 +1,26 @@
+require "pry"
+require "httparty"
+require_relative "recipient"
+
+module SlackAPI
+  class SlackError < StandardError; end
+
+  class User < Recipient
+    BASE_URL = "https://slack.com/api/"
+
+    def self.list_users
+      response = Recipient.get("users.list")
+
+      user_list = []
+
+      if response["ok"]
+        response["members"].each do |member|
+          user_list << {"name" => member["name"], "real name" => member["real_name"], "slack id" => member["id"]}
```

Comment:
This method would make more sense if it returned a list of `User` instances with `name`, `read name` and `id` properties.